config.py

```diff
@@ -14,17 +16,20 @@ class ConfigEmail:
 
 class Config:
     def __init__(self):
+        self.install_path = Path(os.getenv("TINYHIPPO_INSTALL_PATH"))
```

Comment:
How does Path work specifically? And the exporting of environment variables. It seems like the environment variable only exists when tinyHIPPO is running, and otherwise is not. Maybe I need to do more research into how environment variables work, but some clarification would be appreciated

Reply:
The enviornment variable should last for the duration of the programs runtime, and since this variable is set in the start script it should also be reset in the case of a reboot. Path is a library that provides an OS dependent way of navigating file systems by creating a filepath object that can be queried for various properties. It comes built in to python3 so it doesn't create any additional overhead.